rust/hook-worker/src/worker.rs

```diff
@@ -376,15 +378,25 @@ async fn create_hoghook_kafka_payload(
 }
 
 struct WebhookSuccess {
```

Comment:
How do you feel about adding content-type here (and other headers? maybe future work once we know better what hog needs)

Reply:
I think it's a good idea but I will defer it for our first CDP-redesign discussion. If we keep using this, I do think we should return everything you would reasonably expect from a JS `fetch` call. 👍